Mathlib/NumberTheory/AbelSummation.lean

```diff
@@ -0,0 +1,174 @@
+/-
+Copyright (c) 2024 Xavier Roblot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Xavier Roblot
+-/
+import Mathlib.MeasureTheory.Integral.FundThmCalculus
+
+/-!
+# Abel's summation formula
+
+We prove several versions of Abel's summation formula.
+
+## Results
+
+* `sum_mul_eq_sub_sub_integral_mul`: general statement of the formula for a sum between two
+(nonnegative) reals `a` and `b`.
+
+* `sum_mul_eq_sub_integral_mul`: a specialized version of `sum_mul_eq_sub_sub_integral_mul` for
+  the case `a = 0`.
+
+ * `sum_mul_eq_sub_integral_mul`: a specialized version of `sum_mul_eq_sub_integral_mul` for
+  when the first coefficient of the sequence is `0`. This is useful for `ArithmeticFunction`.
+
+## References
+
+* <https://en.wikipedia.org/wiki/Abel%27s_summation_formula>
+
+-/
+
+/-!
+# Docstring
```

Comment:
Comment

Reply:
You've marked this as resolved, but I think Ruben was trying to point out that you have a docstring in your code consisting of the word "docstring" (which is still there).